idaes/gas_solid_contactors/unit_models/tests/test_MB.py

```diff
@@ -307,6 +314,88 @@ def test_conservation(self, iron_oc):
         ebal_tol = ebal_gas + ebal_solid - e_reaction
         assert abs(ebal_tol) <= 1e-2
 
+    @pytest.mark.component
```

Comment:
We will also need a test to check that the model converges with the default scale factors.